src/effects/builtin/whitenoiseeffect.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <random>
+
+#include "effects/effectprocessor.h"
+#include "engine/effects/engineeffect.h"
+#include "engine/effects/engineeffectparameter.h"
+#include "engine/filters/enginefilterpansingle.h"
+#include "util/class.h"
+#include "util/defs.h"
+#include "util/sample.h"
+#include "util/samplebuffer.h"
+#include "util/types.h"
+
+class WhiteNoiseGroupState final : public EffectState {
+  public:
+    WhiteNoiseGroupState(const mixxx::EngineParameters& bufferParameters)
+      : EffectState(bufferParameters),
+        gen(rs()) {
```

Comment:
```suggestion
      : EffectState(bufferParameters),
        previous_drywet(0.0),
        gen(rs()) {
```
When switching between effects there can be a jarring jump in the audio. I think it may be because `previous_drywet` is uninitialised, so it could be any value.